package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "@dydxprotocol/starkex-lib",
-  "version": "1.1.4",
+  "version": "1.1.5",
   "description": "Cryptographic functions for use with StarkEx",
   "main": "build/src/index.js",
   "scripts": {
     "build": "rm -rf build/ && npm run compile && npm run copy-files",
     "compile:watch": "npm run compile -- --watch",
     "compile": "tsc",
     "copy-files": "copyfiles ./**/*.so build/",
-    "coverage": "NODE_ENV=test nyc --reporter=lcovonly mocha 'build/__tests__/**/*.test.js'",
+    "coverage": "NODE_ENV=test nyc --reporter=lcovonly mocha --timeout 4000 'build/__tests__/**/*.test.js'",
```

Comment:
Our jest timeouts are usually custom set to 10k if we start experiencing any timeout on our default jest config. Any reason why this is set to 4000?

Reply:
Updated it to be the same as the "test" script